src/Controls/src/BindingSourceGen/BindingSourceGenerator.cs

```diff
@@ -177,14 +177,12 @@ public override string ToString()
 
 public sealed record MemberAccess(string MemberName) : IPathPart
 {
-	public string PropertyName => MemberName;
-	public bool IsConditional => false;
+	public string? PropertyName => MemberName;
```

Comment:
Im not sure about nullable string here. Now implementations of IPathPart do not share any functionality. It feels like IPathPart could be a class with enum.

Reply:
yeah, we could just `object[] Path` and have a function to do pattern matching to get the property name... but I think we can leave that to some refactoring later (the interface might change again, even though we're getting close to a finish line)

Reply:
Sure, we can handle this later